src/utils/uiUtils.ts

```diff
@@ -5,44 +5,43 @@
 
 "use strict";
 
-import * as opn from "opn";
-import * as vscode from "vscode";
+import { env, MessageItem, OpenDialogOptions, Uri, window, workspace } from "vscode";
 import { terraformChannel } from "../terraformChannel";
 
 export async function openUrlHint(message: string, url: string): Promise<void> {
-    const response = await vscode.window.showInformationMessage(message, DialogOption.learnMore, DialogOption.cancel);
+    const response = await window.showInformationMessage(message, DialogOption.learnMore, DialogOption.cancel);
```

Comment:
Please leave it as it is (`vscode.window`). Let's focus on one task per PR, and do refactoring in another PR.